stable/kommander-thanos/templates/query-stores-configmap.yaml

```diff
@@ -0,0 +1,48 @@
+{{ $ns := .Release.Namespace }}
+{{ $labels := include "kommander-thanos.labels" . | indent 4  }}
+{{ $sa := .Values.kommanderServiceAccount }}
+{{- if .Values.thanos.query.enabled }}
+{{- range .Values.thanos.query.serviceDiscoveryFileConfigMaps }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ . }}
+  namespace: {{ $ns }}
+  labels:
+{{ $labels }}
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: before-hook-creation
+data:
+  stores.yaml: |-
+    - targets: []
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ . }}-configmap-cleanup
+  namespace: {{ $ns }}
+  labels:
+{{ $labels }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-4"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: {{ . }}-configmap-cleanup
+    spec:
+      serviceAccountName: {{ $sa }}
+      containers:
+        - name: kubectl
+          image: docker.io/bitnami/kubectl:1.14.1
```

Comment:
nit (not blocker): Why such an old version of kubectl?

Reply:
I checked what version other jobs were running; they all ran 1.14.1 so I stuck with it also just to be safe!